### PR TITLE
Swift package manager usability

### PR DESCRIPTION
**Pull request by anjusr**

### CODE REVIEW

**Review by phanther — APPROVED** [review]
LGTM

**Review by pjain-sift — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -24,6 +23,48 @@ Through Carthage:
...
+            // Dependencies declare other packages that this package depends on.
+            ​// .package(url: /* package url */, from: "1.0.0"),
+            .​package​(​url​: "https://github.com/SiftScience/sift-ios.git"​, ​from​: "2.0.2"​)
+        ],
+        ​targets​: [
+            ​// Targets are the basic building blocks of a package. A target can define a module or a test suite.
+            ​// Targets can depend on other targets in this package, and on products in packages which this package depends on.
+            ​.target(
+                name: "exampleApp",
+                dependencies: ["sift-ios"]), // Don’t forget to add sift-ios in target dependencies.
```
Comment (phanther) [inline README.md:50]: @anjusr Is this `sift-ios` or `sift_ios`? Given that the import is using `sift_ios`

Comment (anjusr) [reply README.md:50]: Yes the file Package name is sift-ios, so we have to add "sift-ios" as dependencies in Package.swift file. 

But on "import module" any non-alphanumeric characters, they are replaced with an underscore in your product module name.

**Pull request merged**